### ISSUE TITLE
Include co-authors in generated squash merge message

### ISSUE THREAD

**Issue by pkaminski** [Reviewable/Reviewable#935]
Because of the nature of github, when a team of multiple people are involved with a commit only a single person (the one who opened up the pull request which sometimes isn't even a person who worked on it) is the only person mentioned. Inside of github's proper Pull Request system, the add to the bottom of commit messages the following line for each person involved in the commit.

```
Co-authored-by: Fred Rubble <fred@example.com>
Co-authored-by: Jane Doe <jane@example.com>
```

This provides context to the commit for people to understand who all worked on it and such. Currently github does this by saying any author of any commit in that PR is a co-author if not the original creator of the PR. The only thing we have found that could be done better is possible removing a person whose only contribution is merging main into the branch but I'm ok if that isn't done. We just do that because whenever a different PR is completed, we have that person merge their changes into different inflight branches so if any conflicts take place they are better able to work with the inflight branch workers to resolve how the resolution should take place so that the original fixes are not lost but the new efforts are able to still do what they need to do.

To recap, it would be great if your ability to squash and merge could mirror the existing capability of githubs. Right now it looks like it simply leaves off the `Co-authored-by` info and that is a big deal that prevents us from being able to keep everything happening inside of your product.
